docs/08-guides.md

```diff
@@ -189,6 +189,44 @@ import { repeat } from "/web_modules/lit-html/directives/repeat.js";
 // ✔ snowpack installed: lit-html, lit-element [0.25s]
 ```
 
+### Svelvet
+
+[Svelvet](https://github.com/jakedeichert/svelvet) is a cli [svelte](https://github.com/sveltejs/svelte) compiler and file watcher that integrates directly with snowpack.
+
+This component is taken from the example project [found here](https://github.com/jakedeichert/svelvet/tree/master/examples/basic).
+
+```html
+<!-- File: src/App.svelte -->
+<script>
+    import Header from './components/Header';
+    import Footer from './components/Footer/index';
+</script>
+
+<Header />
+<Footer />
+```
+
+And here's the minimum `package.json` you would need:
+
+```json
+{
+  "name": "svelvet-example",
+  "scripts": {
+    "dev": "svelvet",
+    "build": "NODE_ENV=production svelvet"
+  },
+  "dependencies": {
+    "svelte": "^3.7.1"
+  },
+  "devDependencies": {
+    "svelvet": "^0.1.10"
+  }
+}
+```
+
+Use `npm run dev` to compile in watch mode or `npm run build` to generate an optimized, tree-shaken production build. The output is a `dist` directory that contains your compiled svelte app with `web_modules`! For more details, check out the [svelvet guide](https://github.com/jakedeichert/svelvet).
```

Comment:
i've been going back and forth myself on how much documentation is too much, since this could get outdated rather quickly with the version semver, etc. I'd recommend that we replace this package.json with just a simple `npx svelvet` call, cut the first sentence on the different scripts, and then keep the end of this paragraph on what Svelvet generates and where you can go to learn more. Something like:

> Run `npx svelvet` on the command line to start and then use your favorite [local dev server](https://www.snowpack.dev/#installing-a-dev-server) to load your app locally in the browser. Svelvet will write your `src/` application to `dist/` and then watch your source directory for additional changes. For more details and instructions on how to export your Svelte application for production, check out [the full svelvet guide](https://github.com/jakedeichert/svelvet).

Reply:
Thanks for the review! I'm going to update svelvet dev mode to auto run servor this week and then i'll update the docs in this PR :)

Reply:
Released v0.2.0 with servor support https://github.com/jakedeichert/svelvet/releases

Updated the wording here too 👍